views/DeviceListView/components/controls/LightControls.tsx

```diff
@@ -78,40 +80,44 @@ export const LightControls: FC<AdditionalControlsProps> = ({
     });
 
   return (
-    <>
-      <Button
-        text="Włącz wszystkie w całym domu"
-        variant="success"
-        hasFullWidth
-        onPress={handleAllLightsOn}
-      />
-      <Button
-        text="Wyłącz wszystkie w całym domu"
-        variant="error"
-        hasFullWidth
-        onPress={handleAllLightsOff}
-      />
-      <Typography
-        variant="body-medium"
-        text="Wybierz node, którym chcesz zarządzać"
-      />
-      <Select
-        selectData={nodesSelectData}
-        index={selectedIndex}
-        onSelect={onSelect}
-      />
-      <Button
-        text="Włącz w danym nodzie"
-        variant="success"
-        hasFullWidth
-        onPress={handleNodeLightsOn}
-      />
-      <Button
-        text="Wyłącz w danym nodzie"
-        variant="error"
-        hasFullWidth
-        onPress={handleNodeLightsOff}
-      />
-    </>
+    <View style={styles.container}>
+      <View style={styles.content}>
```

Comment:
Why layout isn't applied to this component automatically 🤔

Reply:
Done